ironfish/src/network/peers/localPeer.ts

```diff
@@ -24,6 +24,8 @@ export class LocalPeer {
   readonly version: number
   // constructor for either a Node WebSocket or a browser WebSocket
   readonly webSocket: IsomorphicWebSocketConstructor
+  // asynchronously imported WebRTC datachannel library
+  readonly nodeDataChannel: NodeDataChannelType
```

Comment:
Was there a reason we had this on the localPeer? Guessing it's because the `webSocket` is also here but I couldn't figure out why that was here in the first place either

Reply:
I put it here because `webSocket` was here. I don't necessarily care where it goes, but if we're going to move both of them, I'd like to do it in a separate PR

Reply:
Gotcha yea was just curious. Have no preference for doing it in this PR